clang/runtime/dpct-rt/include/dpl_extras/dpcpp_extensions.h

```diff
@@ -162,12 +162,6 @@ template <int N, int COUNT> struct log2<N, 0, COUNT> {
   enum { VALUE = (1 << (COUNT - 1) < N) ? COUNT : COUNT - 1 };
 };
 
-__dpct_inline__ uint32_t bfe(uint32_t source, uint32_t bit_start,
```

Comment:
Do we need keep backward compatibility here?
Can we keep the interface here, call the ::bfe in its function body and emit a deprecated message to hint user to use ::bfe?

Reply:
This `bfe` defined in the `detail` namespace, I'm not sure should we keep the backward compatibility in `detail` namespace.

Reply:
Oh, you are right. I did not notice that `bfe` is in the `detail` namespace.
I will +1 for this PR.